bentoml_cli/serve.py

```diff
@@ -17,6 +17,30 @@ def add_serve_command(cli: click.Group) -> None:
 
     @cli.command()
     @click.argument("bento", type=click.STRING, default=".")
+    @click.option(
+        "--component",
+        type=click.STRING,
+        default=None,
+        envvar="BENTOML_SERVE_COMPONENT",
+        help="[Experimental] Component (`api-server` or `runner`) to serve, if not specified, will serve all components",
```

Comment:
Should it be `api_server` with a underscore? `api_server` is how it appears in other references.